src/main/rx/cc2500_frsky_x.c

```diff
@@ -186,6 +186,8 @@ static void buildTelemetryFrame(uint8_t *packet)
     case RX_SPI_FRSKY_X_LBT_V2:
         frame[3] = rxCc2500SpiConfig()->bindTxId[2];
         break;
+    default:
+        break;
```

Comment:
`frame[3]` will not be set to anything. Don't know if that's bad or not.

Reply:
Don't have the hardware to test this but it should not hit the default anyway.

Reply:
Changed to if/else to remove the default